canvas_modules/common-canvas/src/common-properties/components/editor-form/editor-form.jsx

```diff
@@ -89,6 +89,15 @@ class EditorForm extends React.Component {
 		return true;
 	}
 
+	componentDidUpdate() {
+		// Scroll to the selected accordion even when clicked from Alerts tab
+		const activeTabId = this.props.activeTab;
```

Comment:
Should this only be set when the activeTab changes?  There probably should be a check that compares the old/new props to make sure the activeTab changed.  If it's different then find the element and scroll to it.

Reply:
> Should this only be set when the activeTab changes? There probably should be a check that compares the old/new props to make sure the activeTab changed. If it's different then find the element and scroll to it.

Hi @matthoward366 since we are enabling multiple tabs to be open at once so there are multiple `activeTab` values. In below scenario if I click on Alert it will scroll to summary-panel but if I click on Alert again it will not scroll in cases if we check props since both are summary-panel.

https://github.com/elyra-ai/canvas/assets/16302714/574517b5-0349-46d4-98af-46c91b5c2846

Reply:
I understand it works fine but in a `componentDidUpdate` update call usually you want to check if the current props and old props are different before setting something.  I think wrapping this in an if check makes things safer and wouldn't be setting `activeTabId` each time componentDidUpdate is called which is every time there is a prop or state update.

Reply:
Sure Matt. I have updated PR with a condition to check props and check if the alert link is clicked, please let me know if I need to follow any other approach.